pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -150,13 +149,10 @@ type ThanosConfig struct {
 	//
 	// +optional
 	Chart *ChartConfig `json:"chart,omitempty"`
-	// Mode defines the mode of the Thanos.
-	// The default is sidecar.
-	// +kubebuilder:validation:Enum=sidecar;reciever
-	// +optional
-	Mode string `json:"mode,omitempty"`
-	// ObjectStoreConfig defines the configuration for the object store.
-	ObjectStore *ThanosObjectStoreConfig `json:"objectStore,omitempty"`
+	// ObjstoreSecretRef is the secret reference of the object store.
+	// Must follow the definition of the thanos: https://thanos.io/tip/thanos/storage.md/.
+	// +required
+	ObjstoreSecretRef string `json:"objstoreSecretRef"`
```

Comment:
```suggestion
	ObjStoreSecretRef string `json:"objstoreSecretRef"`
```

Reply:
`objestore` came from project https://github.com/thanos-io/objstore, is should be `Objstore` or `ObjectStore`?

Reply:
Store `s` should be uppercase